project/scripts/sbt

```diff
@@ -3,20 +3,17 @@
 # Usage:
 # ./sbt <cmd>
 
-set -e
-
 CMD="$1"
 
 if [ -z "$CMD" ]; then
   echo "Error: missing sbt command"
   exit 1
 fi
 
-# get the ivy2 cache
-ln -s /var/cache/drone/ivy2 "$HOME/.ivy2"
-
 # run sbt with the supplied arg
 sbt -J-Xmx4096m \
     -J-XX:ReservedCodeCacheSize=512m \
     -J-XX:MaxMetaspaceSize=1024m \
-    -Ddotty.drone.mem=4096m "$CMD"
+    -Ddotty.drone.mem=4096m \
+    -ivy /var/cache/drone/ivy2 \
```

Comment:
I this doesn't do what you think it does. This way you still download all the dependence needed for sbt itself...

Reply:
No sure about that. If you look at the CI test logs (except for test_bootstrapped), I don't see any download.